[2024-03-27T20:00:36.741+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T20:00:36.837+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T11:00:00+00:00 [queued]>
[2024-03-27T20:00:36.858+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T11:00:00+00:00 [queued]>
[2024-03-27T20:00:36.858+0800] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-03-27T20:00:36.884+0800] {taskinstance.py:1327} INFO - Executing <Task(BigQueryDeleteTableOperator): delete_daily_bigquery_external_table_task> on 2024-03-27 11:00:00+00:00
[2024-03-27T20:00:36.895+0800] {standard_task_runner.py:57} INFO - Started process 22699 to run task
[2024-03-27T20:00:36.907+0800] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'update_daily_table_test45', 'delete_daily_bigquery_external_table_task', 'scheduled__2024-03-27T11:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/flights_ingestion_dag.py', '--cfg-path', '/tmp/tmpn8op65zg']
[2024-03-27T20:00:36.912+0800] {standard_task_runner.py:85} INFO - Job 25: Subtask delete_daily_bigquery_external_table_task
[2024-03-27T20:00:36.943+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T20:00:37.022+0800] {task_command.py:410} INFO - Running <TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T11:00:00+00:00 [running]> on host 2ed6da3b8fe0
[2024-03-27T20:00:37.043+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T20:00:37.182+0800] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_daily_table_test45' AIRFLOW_CTX_TASK_ID='delete_daily_bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T11:00:00+00:00'
[2024-03-27T20:00:37.184+0800] {bigquery.py:2348} INFO - Deleting: flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T20:00:37.186+0800] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-27T20:00:37.484+0800] {bigquery.py:1231} INFO - Deleted table flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T20:00:37.500+0800] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=update_daily_table_test45, task_id=delete_daily_bigquery_external_table_task, execution_date=20240327T110000, start_date=20240327T120036, end_date=20240327T120037
[2024-03-27T20:00:37.557+0800] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-03-27T20:00:37.593+0800] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
